block_pickup.php

```diff
@@ -39,8 +41,8 @@ public function init() {
      *
      */
     public function specialization() {
-        if (isset($this->config->title)) {
-            $this->title = $this->title = format_string($this->config->title, true, ['context' => $this->context]);
+        if (!empty($this->config->title)) {
```

Comment:
More accurate to use empty() to check for empty string

Reply:
No title ever on this block.